pyproject.toml

```diff
@@ -90,3 +90,13 @@ module = [
 ignore_errors = true
 warn_unused_ignores = false
 
+
+[tool.cibuildwheel]
+before-test = ""
+test-requires = "pytest pytest-xdist"
+# Need to change root directory, otherisw the plugin won't find setup.cfg
+test-command = "(cd {project}; pytest --trace-config;  pytest -n2 -q test --nomemory --notimingintensive)"
```

Comment:
`pytest --trace-config;` should be removed

Reply:
that seems to be the case, since their example at https://cibuildwheel.readthedocs.io/en/stable/options/#environment-variables has ` CIBW_TEST_COMMAND: "pytest {project}/tests"`

I guess if using cd is working fine then that's fine

Reply:
Bandaid fix, I'm sure there's something better ([since windows doesn't like it](https://github.com/sqlalchemy/sqlalchemy/runs/6323450825?check_suite_focus=true#step:3:200))but I just don't know what :)

Again going to numpy for a possible example, they run the test suite from within python which I guess resolves those errors. Is that currently doable for sqla?

https://github.com/numpy/numpy/blob/main/tools/wheels/cibw_test_command.sh They also check the license, if that's of any importance

Reply:
we can take a look. having a sh script would be fine, sure. Also it seems you can have different commands for windows vs linux, so we can custimize the test to suit windows https://cibuildwheel.readthedocs.io/en/stable/options/#test-command